pkg/arvo/sys/vane/gall.hoon

```diff
@@ -699,11 +702,12 @@
   ::  +mo-load: install agents
   ::
   ++  mo-load
-    |=  agents=(list [=dude =beak =agent])
+    |=  agents=(list [=dude =beak =agent-any])
     =.  mo-core
       |-  ^+  mo-core
       ?~  agents  mo-core
-      =/  [=dude =desk]  [dude q.beak]:i.agents
+      =/  [=dude =desk =agent-any]  [dude q.beak agent-any]:i.agents
+      ?>  ?=([%zuse %416] -.agent-any)
```

Comment:
Currently just asserting this everywhere, the `agent-any` type allows us to build shims in the future.